### DISCUSSION TITLE
Localization and collaboration with Transifex

### DISCUSSION THREAD

**Discussion by oittaa** [teslamate-org/teslamate#604] [Ideas]
I'm interested in starting a Finnish translation for Teslamate, but the effort of translating everything alone is quite daunting. Would it make sense to centralize translation efforts with proper collaboration tools? Transifex is free for Open Source projects.

https://docs.transifex.com/projects/open-source-project

Edit: For example Signal Messenger uses Transifex.

**Comment by adriankumpf** [comment]
I haven't heard of Transifex. I'm going to check it out.

To start a new translation, generally these two files would have to be translated (the first file is much more important):

* [default.pot]( https://github.com/adriankumpf/teslamate/blob/master/priv/gettext/default.pot)
* [errors.pot](https://github.com/adriankumpf/teslamate/blob/master/priv/gettext/errors.pot)

**Comment by oittaa** [comment]
Cool! Transifex, or some other similar tool, would make it super easy for non-programmers to contribute.

**Comment by SiNaPsEr0x** [comment]
But is it possible to edit the fields created on grafana instead?
